tests/start-server.sh

```diff
@@ -9,8 +9,8 @@ mkdir -p /tmp/certs
 cp ./tests/public.crt ./tests/private.key /tmp/certs/
 
 (MINIO_CI_CD=true \
-    MINIO_NOTIFY_WEBHOOK_ENABLE_miniojavatest=on \
-    MINIO_NOTIFY_WEBHOOK_ENDPOINT_miniojavatest=http://example.org/ \
+    MINIO_NOTIFY_WEBHOOK_ENABLE_miniorustest=on \
+    MINIO_NOTIFY_WEBHOOK_ENDPOINT_miniorustest=http://example.org/ \
```

Comment:
This will not work in our `mint` testing i.e. the test server does not have this webhook and notification target. You could leave as is here and below `tests/tests.rs`

Reply:
> How you run your S3 server/service for local testing is out of scope of this project

Nope. It's not out of scope. It's extremely in the scope

> You cannot use Github CI script locally

But I can copy commands from there and emulate github actions

>  In fact, there is no need to test minio-rs locally

New features? Fixing/changing/replacing internal methods? As a contributor I would like to have an easy tool to test new changes. Furthermore, it's just 3 files. It reduces contributors time spent on introduction to repository and code testing by the large factor. It's also saving time of the minio team - you don't have to spend time allowing workflows that will obviously fail

> If a contributor does that, it is better to add respective functional tests rather than testing locally.

How does one connect to another in this case?